app/db_models.py

```diff
@@ -345,6 +347,64 @@ def update_obj(self, question, data):
             if field in data:
                 setattr(question, field, data[field])
 
+class FillInTheBlankQuestion(Question):
+    id = db.Column(db.Integer, db.ForeignKey('question.id'), primary_key=True)
+
+    question_text = db.Column(db.String, nullable=False)
+
+    answer = db.Column(db.String, nullable=False)
+
+    __mapper_args__ = {
+        'polymorphic_identity': QuestionType.FILL_IN_THE_BLANK_QUESTION,
+    }
+
+    def get_answer(self):
+        return markdown_to_html(self.answer)
+    
+    def make_question(question_text):
```

Comment:
I'm not sure the role of this function. Before writing your function, always write the docstring comment to explain what the function will do.